tests/integration/client/feedback/dataset/remote/test_list_dataset_records.py

```diff
@@ -0,0 +1,70 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import argilla as rg
+import pytest
+from argilla import User, ValueSchema, Workspace
+from argilla.client.feedback.schemas.enums import ResponseStatus
+from argilla.client.sdk.users.models import UserRole
+from argilla.feedback import FeedbackDataset, FeedbackRecord, ResponseSchema, TextField, TextQuestion
+
+
+class TestSuiteListDatasetRecords:
```

Comment:
I think that adding `Suite` is not really useful: 

```suggestion
class TestListDatasetRecords:
```